flight_paths.py

```diff
@@ -0,0 +1,89 @@
+import json
+from simple_graph import Node, SimpleGraph
+from simple_graph import spt_Dijkstra
+
+
+def extract_data_from_json(file):
+    with open(file) as json_data:
+        data = json.load(json_data)
+    new_data = []
+    for d in data:
+        entry = {}
+        entry['city'] = d['city']
+        entry['neighbors'] = d['destination_cities']
+        entry['lat-lon'] = d['lat_lon']
+        new_data.append(entry)
+    return new_data
+
+
+def build_graph(new_data):
+    gr = SimpleGraph()
+
+    for city in new_data:
+        n = Node(city['city'], city['lat-lon'])
+        try:
+            gr.add_node(n)
+        except KeyError:
+            pass
+
+    for city in new_data:
+        if city['city'] not in gr.node_dict.keys():
+            continue
+        key = city['city']
+        for neighbor in city['neighbors']:
```

Comment:
It's interesting that you'd add neighbors here and not in the previous loop, but I suppose you wanted to build your neighbors list with the distances between cities already baked in.